signup/signup.php

```diff
@@ -0,0 +1,314 @@
+			<li class="form-textfield">
+				<label class="form-label form-required" for="firstName">First Name</label>
+				<input type="text" name="firstName" id="firstName" ng-model="data.firstName" required>
+				<div ng-show="form.$submitted || form.firstName.$touched">
+					<label class="error" ng-show="form.firstName.$error.required">This field is required.</label>
+				</div>
+			</li>
+
+			<li class="form-textfield">
+				<label class="form-label form-required" for="lastName">Last Name</label>
+				<input type="text" name="lastName" id="lastName" ng-model="data.lastName" required>
+				<div ng-show="form.$submitted || form.lastName.$touched">
+					<label class="error" ng-show="form.lastName.$error.required">This field is required.</label>
+				</div>
+			</li>
+
+			<li class="form-textfield">
+				<label class="form-label" for="email">Email</label>
+				<input type="email" name="email" id="email" ng-model="data.email">
+			</li>
+
+			<li class="form-textfield">
+				<label class="form-label form-required" for="phone">Phone Number</label>
+				<input type="text" name="phone" id="phone" ng-model="data.phone" required>
+				<div ng-show="form.$submitted || form.phone.$touched">
+					<label class="error" ng-show="form.phone.$error.required">This field is required.</label>
+				</div>
+			</li>
+		</ul>
+
+
+		<h3 class="form-subheading">Add Filing Dependents</h3>
+		<p>Are any of your dependents filing a return during this appointment? If so, add them here.</p>
+		<ul id="dependents">
+			<li class="wdn-grid-set dependent" ng-repeat="dependent in dependents">
+				<ng-form name="innerForm" class="flex">
+					<div class="bp768-wdn-col-two-fifths">
+						<label for="firstNameInput1" class="form-label form-required">First Name</label>
+						<input class="firstName" type="text" name="dependentFirstName" ng-model="dependent.firstName" required>
+						<div ng-show="form.$submitted || innerForm.dependentFirstName.$touched">
+							<label class="error" ng-show="innerForm.dependentFirstName.$error.required">This field is required.</label>
+						</div>
+					</div>
+					<div class="bp768-wdn-col-two-fifths">
+						<label for="lastNameInput1" class="form-label form-required">Last Name</label>
+						<input class="lastName" type="text" name="dependentLastName" ng-model="dependent.lastName" required>
+						<div ng-show="form.$submitted || innerForm.dependentLastName.$touched">
+							<label class="error" ng-show="innerForm.dependentLastName.$error.required">This field is required.</label>
+						</div>
+					</div>
+					<div class="bp768-wdn-col-one-fifth removeDependentButton px-1rem">
+						<button type="button" class="wdn-button wdn-button-brand" ng-click="removeDependent(dependent)">Remove Dependent</button>
+					</div>
+				</ng-form>
+			</li>
+		</ul>
+		<br>
+		<button type="button" class="wdn-button wdn-button-triad" id="addDependentButton" ng-click="addDependent()">Add Dependent</button>
+
+		<style>
+		@media (min-width: 768px) {
```

Comment:
Should this get pulled out to the signup.css file?

Reply:
yes it probably should?